src/Gridify/Reflection/MemberNullPropagatorVisitor.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq.Expressions;
+
+namespace Gridify.Reflection;
+
+//https://stackoverflow.com/a/30489160/1698682
+public class MemberNullPropogationVisitor : ExpressionVisitor
+{
+   protected override Expression VisitMember(MemberExpression node)
+   {
+      if (node.Expression == null || !IsNullable(node.Expression.Type))
+         return base.VisitMember(node);
+
+      if (node.Type.IsValueType && node.Type != typeof(Nullable<>))
+      {
+         return base.VisitMember(node);
+      }
+      else
+      {
+         var expression = base.Visit(node.Expression);
+         var nullBaseExpression = Expression.Constant(null, expression.Type);
+         var test = Expression.Equal(expression, nullBaseExpression);
+         var memberAccess = Expression.MakeMemberAccess(expression, node.Member);
```

Comment:
This is not used. maybe use `_ = Expression.MakeMemberAccess(expression, node.Member); `  if we need this call